gitops/overlays/prod/configs/frontend/config.conf

```diff
@@ -0,0 +1,51 @@
+SCCH_BASE_URI=https://service.canada.ca
+MSCA_BASE_URI=https://srv136.services.gc.ca
```

Comment:
These values are the default values that are set in the application - no need to explicitly set them in this file.

Reply:
Fixed that.